dev/TabView/TabView.cpp

```diff
@@ -68,6 +68,16 @@ TabView::TabView()
     }
 }
 
+TabView::~TabView()
+{
+    UnhookEventsAndClearFields();
+    if (const auto footer = TabStripFooter().try_as<winrt::FrameworkElement>())
+    {
+        footer.UnregisterPropertyChangedCallback(winrt::FrameworkElement::MinWidthProperty(), m_footerMinWidthProperyChangedToken);
```

Comment:
>m_footerMinWidthProperyChangedToken [](start = 94, length = 35)

Do we need to check to see if this token is not 0?

Reply:
I don't think it can be 0 if we actually get in there since that would require us to have a footer, yet not having registered to said footer. Anytime the footer changes, we update the tokens so we would need to get here with footer changes being unnoticed by the TabView.